tests/integration/long_callback/app_progress_delete.py

```diff
@@ -35,9 +36,12 @@
     prevent_initial_call=True,
 )
 def on_bg_progress(set_progress, _):
-    set_progress("start")
-    time.sleep(2)
-    set_progress("stop")
+    with lock:
+        set_progress("start")
+        time.sleep(0.5)
```

Comment:
should we pull `0.5` out into a setting somewhere, or is it unlikely to ever change?